tests/Makefile

```diff
@@ -132,8 +132,14 @@ dirs: $(DIRS)
 $(DIRS):
 	$(MAKE) -C $@
 
+ifdef MYST_REMOVE_APPDIRS
+remove_appdir = rm -rf $(1)/appdir
+else
+remove_appdir =
+endif
+
 __tests:
-	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) tests $(NL) )
+	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) tests; $(call remove_appdir,$(i)) $(NL) )
```

Comment:
Can you add the additional step to `solutions` as well?

Reply:
I don't think this is going to work, which is why it is experimental. I will update soon.

Reply:
Okay, it seems to work and I addressed your request.